src/test/resources/features/Login.feature

```diff
@@ -0,0 +1,10 @@
+Feature: Login in an account
+
+  Scenario: login in trabajopolis
```

Comment:
Please use capital letter at the beginning

Reply:
done